os_stub/spdm_device_secret_lib_sample/csr.c

```diff
@@ -327,7 +335,9 @@ bool libspdm_gen_csr_ex(
         if (*req_csr_tracking_tag == 0) {
             if (available_csr_tracking_tag == 0) {
                 /*no available tracking tag*/
-                *req_csr_tracking_tag = 0xFF;
+                #if LIBSPDM_SET_CERT_CSR_PARAMS
+                *is_busy = true;
```

Comment:
The comment above said "no available tracking tag" so I am confused how does this imply "device is busy".
Would you help clarify this?

Reply:
That comes from 
> If the Responder requires a reset to process a GET_CSR request, but does not have any available CSRTrackingTags, it shall respond with an ERROR message of ErrorCode=Busy.